puppet/Puppetfile

```diff
@@ -8,6 +8,7 @@ mod 'puppet/augeasproviders_ssh', '7.0.0'
 mod 'puppet/chrony', '4.0.0'
 mod 'puppet/epel', '5.0.0'
 mod 'puppet/extlib', '7.5.1'
+mod 'puppet/grafana_alloy', :git => 'https://github.com/voxpupuli/puppet-grafana_alloy', :branch => 'main'
```

Comment:
Can we create a release?

Reply:
Absolutely. I was just poking things and didn't want to release before I actually tried to use it. By now things should be fine.